docs/content/index.mdx

```diff
@@ -9,7 +9,7 @@ type: Docs
 
 - There are two different variants of this plugin:
 
-  - [`next-pwa`](/next-pwa/getting-started)
+  - [`next-pwa`](/docs/next-pwa/getting-started)
   - [`next-sw` (not stable)](/next-sw)
```

Comment:
```suggestion
  - [`next-sw` (not stable)](/docs/next-sw)
```
Also fix this link please :) I mean, it's most likely that noone actually clicks it, but it'd still be nice to have it fixed.